### ISSUE TITLE
Let's beat the state of the art on NER once again! 

### ISSUE THREAD

**Issue by LifeIsStrange** [Adaxry/GCDT#4]
Hi Adaxry! 

You have used BERT large which at the time was the state of the art pre trained transformer.
But nowadays is had been succeeded by XLnet which is a major improvement. 
Very often switching from BERT large to XLnet large give multiple percents accuracy gains "for free"! 

https://paperswithcode.com/paper/xlnet-generalized-autoregressive-pretraining
You could either use the official implementation (not actively maintained) or the one maintained by tensorflow. 
But I believe the most actively maintained is 
https://github.com/huggingface/transformers which allow you through the same API to switch to another transformer easily. 

I and Humanity really needs more accurate named entity recognition! :m
